webbpsf/optical_budget.py

```diff
@@ -359,7 +361,7 @@ def visualize_wfe_budget(inst, slew_delta_time=14 * u.day, slew_case='EOL', ptt_
         annotate_budget='Image motion (as equiv. WFE)',
         instname=inst.name,
     )
-    ax.text(0.04, 0.92, f'LOS jitter: {jitter}, 1$\sigma$/axis', transform=ax.transAxes)
+    ax.text(0.04, 0.92, f'LOS jitter: {jitter}, 1$\sigma$/axis', transform=ax.transAxes) # noqa sigma not escape char
```

Comment:
Same as above, use `\\` for a literal backslash.

Reply:
expanding, I thought these were latex specific symbols.